### PR TITLE
framework/st_things : If initialize fail exception handling at start

### PR DESCRIPTION
**Pull request by JeongJunSik**
- Change the location where Wifi turns on (at startup -> after creating the resource)
- If initialize fails, exception handling at start
- Remove useless code(g_quit_flag)
- Remove the variable that was being used incorrectly(is_there_module_inited)

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [67336407e787dc417887acbebb493112197ef34b] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [67336407e787dc417887acbebb493112197ef34b] - Code Rule Check Result:
framework/src/st_things/things_stack/things_stack.c:293: ERROR: [SPC_M_KWD] space required before the open brace '{'

**Comment by seinfra** [comment]
Target : [67336407e787dc417887acbebb493112197ef34b] -

**Comment by Hyunjun85** [comment]
Plz, check code rule.

**Comment by JeongJunSik** [comment]
Ok, i fixed code rule.

**Comment by seinfra** [comment]
Target : [05f549f6902feab1d8e98eb4872661748aabce0a] -

**Comment by seinfra** [comment]
Target : [05f549f6902feab1d8e98eb4872661748aabce0a] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [05f549f6902feab1d8e98eb4872661748aabce0a] - Code Rule Check (C++) OK.

File: framework/src/st_things/things_stack/things_stack.c [hunk 1]
```diff
@@ -293,22 +290,12 @@ int things_start_stack(void)
...
-			}
-		} else if (!things_network_connect_home_ap()) {
-			return 0;
-		}
-	} else if (dm_get_easysetup_connectivity_type() == es_conn_type_ble) {
-		//TO DO
-	} else {
-		THINGS_LOG_E(TAG, "es_conn_type_Unknown");
+	if (!is_things_module_inited) {
+		THINGS_LOG_E(TAG, "Initialize failed. You must initialize it first.");
```
Comment (juitem) [inline framework/src/st_things/things_stack/things_stack.c]: how about change _inited -> _initialized

Comment (JeongJunSik) [reply framework/src/st_things/things_stack/things_stack.c]: Ok.

**Comment by seinfra** [comment]
Target : [d7dd775c3a98ee6159a965a31268299b34989bac] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [d7dd775c3a98ee6159a965a31268299b34989bac] - Code Rule Check OK.

**Comment by juitem** [comment]
would you please check travis error?
2943/root/tizenrt/framework/src/st_things/things_stack/utils/things_network.c:285: undefined reference to `things_is_things_module_inited'

**Comment by seinfra** [comment]
Target : [14f167f31c1268a17dfbdf872f98b5d90c4ebb20] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [14f167f31c1268a17dfbdf872f98b5d90c4ebb20] - Code Rule Check OK.

**Pull request merged**